tools/travis/install_openwhisk.sh

```diff
@@ -4,7 +4,7 @@ HOMEDIR="$(dirname "$TRAVIS_BUILD_DIR")"
 cd $HOMEDIR
 
 # OpenWhisk stuff
-git clone --depth 3 https://github.com/openwhisk/openwhisk.git
+git clone --depth 3 https://github.com/apache/incubator-openwhisk.git
```

Comment:
to preserve paths to an "/openwhisk" do we want to use:
git clone --depth 3 https://github.com/apache/incubator-openwhisk.git openwhisk

to not create an "incubator-openwhisk" directory which might cause other issues?